webapps/assembly/src/main/java/org/camunda/bpm/webapp/impl/security/auth/UserAuthenticationResource.java

```diff
@@ -166,4 +181,8 @@ protected Response forbidden() {
   protected Response notFound() {
     return Response.status(Status.NOT_FOUND).build();
   }
+
+  private boolean isWebappsAuthenticationLoggingEnabled(ProcessEngine processEngine) {
+    return ((ProcessEngineConfigurationImpl) processEngine.getProcessEngineConfiguration()).isWebappsAuthenticationLoggingEnabled();
```

Comment:
One benefit of not adding a flag immediately to the interface, which is part of the public API, is that the flag could be easily removed in the future. Not saying that we have any praxis around that, but it would be potentially possible.

Reply:
I don't think there is a good reason to not do the cast in this situation. I figured since this feature is not intended for use via API in other projects, only "internally" for our web apps, I would place the property in the `Impl` class and not make it public API.

Let me know if any of you have a strong opinion on why we should move it into the interface and I will do it. Otherwise, I would vote to keep it in the implementation class.